Dockerfile.dev

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17
```

Comment:
This image is 1.31GB ... and cleaning the cache doesn't help.

 It seems that it is due to the go install which has the stdlib precomipiled a bunch.

Do we really need it once the tools below are build? Can we maybe copy it to a different, smaller image. All the tools below are under 100mb.

Reply:
Well, to be honest, we don't need all tools installed, but to be honest, the size of the tools container is not so important. It's not the image that pulls in the CI during the deployment, right? 

I'd say that the more important is the maintainability. So fewer instructions it has is better :man_shrugging: 

Not sure that here applies the same approach as we do in the main image since, for example, for the lining, we need the binary and go installed. But let me try to iterate on this. Maybe we can decrease it :thinking:

Reply:
So, migrated to the alpine (it's also consistent with the main `Dockerfile`), the size is decreased but still significant. So maybe we can iterate on it in the following PRs? In any case, I hope that this will be a team's effort so it can be constantly improved. WDYT?

```bash
docker image ls | grep k6-dev-tools
k6-dev-tools                      latest          0787216b5245   About an hour ago   888MB
```